bindings/node.js/README.md

```diff
@@ -1,93 +1,146 @@
-This package wraps the c-kzg C code in C++ NAPI bindings which allow it to be imported into a NodeJS program.
+# C-KZG-4844
+
+This is a TypeScript library for EIP-4844 that implements the [Polynomial
+Commitments](https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md)
+API. The core functionality was originally a stripped-down copy of
+[C-KZG](https://github.com/benjaminion/c-kzg), but has been heavily modified
+since then. This package wraps that native `c-kzg` C code in C/C++ NAPI bindings for use in node.js applications.
 
 Spec: https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/polynomial-commitments.md
 
-# Usage
+## Prerequisites
+
+Installation requires compilation of C code. Target environment must have:
 
-Install this library with
+- A compiler like g++ or clang
+- [`make`](https://www.gnu.org/software/make/manual/make.html)
+- [`python3`](https://docs.python.org/3/)
+
+## Installation
 
 ```sh
 yarn add c-kzg
+# or
+npm i -S c-kzg
 ```
 
-Import from it like any other library
+## Usage
 
-```js
+```ts
 import {
-  KZGCommitment,
+  BYTES_PER_BLOB,
+  Blob,
+  Bytes48,
   blobToKzgCommitment,
-  verifyAggregateKzgProof,
-  loadTrustedSetup,
-  transformTrustedSetupJSON,
+  computeBlobKzgProof,
+  verifyBlobKzgProofBatch,
 } from "c-kzg";
-```
-
-# Requirements
-
-The C and C++ code is compiled by node-gyp on installation. Your environment will need
-
-- A compiler like g++ or clang
-- `make`
-- `python3`
-
-# Contributing
 
-This directory contains the code necessary to generate NodeJS bindings for C-KZG.
-
-The bindings file has the following interface:
-
-```js
-
-  loadTrustedSetup: (filePath: string) => SetupHandle;
-
-  freeTrustedSetup: (setupHandle: SetupHandle) => void;
-
-  blobToKzgCommitment: (blob: Blob, setupHandle: SetupHandle) => KZGCommitment;
-
-  computeAggregateKzgProof: (
-    blobs: Blob[],
-    setupHandle: SetupHandle
-  ) => KZGProof;
-
-  verifyAggregateKzgProof: (
-    blobs: Blob[],
-    expectedKzgCommitments: KZGCommitment[],
-    kzgAggregatedProof: KZGProof,
-    setupHandle: SetupHandle
-  ) => boolean;
+const blobs = [] as Blob[];
+const commitments = [] as Bytes48[];
+const proofs = [] as Bytes48[];
+
+for (let i = 0; i < BATCH_SIZE; i++) {
+  blobs.push(
+    Buffer.alloc(BYTES_PER_BLOB, "*"));
+  commitments.push(
+    blobToKzgCommitment(blobs[i]));
+  proofs.push(
+    computeBlobKzgProof(blobs[i], commitments[i]));
```

Comment:
I think this would look a lot better without the line breaks.
```suggestion
  blobs.push(Buffer.alloc(BYTES_PER_BLOB, "*"));
  commitments.push(blobToKzgCommitment(blobs[i]));
  proofs.push(computeBlobKzgProof(blobs[i], commitments[i]));
```
Also, I understand that correctly, I'm fairly certain that will be an invalid blob. What does the `*` mean?

Reply:
Ok. Updated.